### PR TITLE
Changes related to the next MeiliSearch release (v0.25.0)

### PR DESCRIPTION
**Pull request by bidoubiwa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,17 +130,19 @@ If something goes wrong during the version update process a rollback occurs:
...
 
-The migration fails for whatever reason. The script uses the cached `v0.21.0` binary and the cached `data.ms` of the previous version to rollback to its original state.
+The migration fails for whatever reason. The script uses the cached `v0.23.0` binary and the cached `data.ms` of the previous version to rollback to its original state.
 
 ## 💔 Version incompatibilities
 
-Incompatibility between versions happens in some specific cases. The breaking changes should be described in the CHANGELOG of the release.
+Versions that are lower than the v0.22.0 can not be migrated. 
+
+It may also happen that versions are incompatibility between each other in some specific cases. The breaking changes should be described in the CHANGELOG of the release.
```
Comment (curquiza) [inline README.md:145]: Sorry, I remember I already said I did not understand this sentence and you explained to me, but I don't remember, I don't understand what we mean here again 😂

Comment (bidoubiwa) [reply README.md:145]: > To understand why the versions are incompatible the changes should be described in the CHANGELOG of the release in the [meilisearch release](https://github.com/meilisearch/MeiliSearch/releases).

Do you prefer this?

Comment (curquiza) [reply README.md:145]: Why "should be described"? Do you mean the description of the changes is available in the changlogs of the release? The "should" looks like the user has to do something

**Review by alallema — APPROVED** [review]
LGTM! 🍳

**Pull request merged**